stdlib/2.7/codecs.pyi

```diff
@@ -115,9 +115,9 @@ class Codec:
         ...
 
 class IncrementalEncoder:
+    errors = ...  # type: str
     def __init__(self, errors: str = ...) -> None:
-        self.errors = errors
-    @abstractmethod
```

Comment:
main reason was pytype compatibility -- the way I justified it to myself was that the implementation doesn't use the actual abc.abstractmethod, but I can see how that's debatable...